scipy/ndimage/filters.py

```diff
@@ -274,6 +305,9 @@ def gaussian_filter(input, sigma, order=0, output=None,
     truncate : float
         Truncate the filter at this many standard deviations.
         Default is 4.0.
+    weights : ndarray
```

Comment:
I'd rather not add this argument. By doing this it basically (optionally) turns this function into correlate1d with a more complex signature. It would probably be better just to use correlate1d directly in that use case.